bundles/org.eclipse.chronograph.api/src/org/eclipse/chronograph/api/adapters/BrickAdapter.java

```diff
@@ -0,0 +1,24 @@
+/*******************************************************************************
+ *	Copyright (c) 2020 ArSysOp
+ *
+ *	This program and the accompanying materials are made available under the
+ *	terms of the Eclipse Public License 2.0 which is available at
+ *	http://www.eclipse.org/legal/epl-2.0.
+ *
+ *	SPDX-License-Identifier: EPL-2.0
+ *
+ *	Contributors:
+ *	Sergei Kovalchuk <sergei.kovalchuk@arsysop.ru> - 
+ *												initial API and implementation
+ *******************************************************************************/
+package org.eclipse.chronograph.api.adapters;
+
+import java.util.List;
+
+import org.eclipse.chronograph.api.Group;
+
+public interface BrickAdapter<T, S> {
```

Comment:
javadoc: what it for, info about template types

Reply:
updated